README.md

```diff
@@ -20,7 +20,8 @@ repositories {
     mavenCentral()
 }
 dependencies {
-    compile 'io.keen:keen-client-api-java:5.2.0'
+    compile 'io.keen:keen-client-api-java:5.2.0'  // streaming 
+    compile 'io.keen:keen-client-api-query:5.2.0' // querying
```

Comment:
This is only needed for query functionality, which is discussed here: https://github.com/keenlabs/KeenClient-Java/blob/35ae90dfbf4e9a02b9d60693f92bf18d3a9711bb/README.md#add-the-keen-query-client-package-to-your-build
So, I'm not sure if it should be placed in this section or not.

Reply:
Hi @masojus - thanks for the feedback. My intention was to emphasise that `io.keen:keen-client-api-query:5.2.0` is different dependency than `io.keen:keen-client-api-java:5.2.0`. Basing on your comment I moved it to a proper section of readme